src/logic/tokens/utils/tokenHelpers.ts

```diff
@@ -71,55 +65,21 @@ export const getERC721Symbol = async (contractAddress: string): Promise<string>
   return tokenSymbol
 }
 
-export const getERC20DecimalsAndSymbol = async (
-  tokenAddress: string,
-): Promise<{ decimals: number; symbol: string }> => {
-  const tokenInfo = { decimals: 18, symbol: 'UNKNOWN' }
-  try {
-    const storedTokenInfo = await getTokenInfos(tokenAddress)
-
-    if (!storedTokenInfo) {
-      const [tokenDecimals, tokenSymbol] = await generateBatchRequests({
-        abi: ALTERNATIVE_TOKEN_ABI,
-        address: tokenAddress,
-        methods: ['decimals', 'symbol'],
-      })
-      return { decimals: Number(tokenDecimals), symbol: tokenSymbol }
-    }
-    return { decimals: storedTokenInfo.decimals as number, symbol: storedTokenInfo.symbol }
-  } catch (err) {
-    console.error(`Failed to retrieve token info for ERC20 token ${tokenAddress}`)
-  }
-
-  return tokenInfo
-}
-
 export const isSendERC20Transaction = async (
   tx: TxServiceModel,
   txCode: string | null,
   knownTokens: Map<string, Token>,
 ): Promise<boolean> => {
-  let isSendTokenTx = !isSendERC721Transaction(tx, txCode, knownTokens) && isTokenTransfer(tx)
-
-  if (isSendTokenTx) {
-    const { decimals, symbol } = await getERC20DecimalsAndSymbol(tx.to)
-
-    // some contracts may implement the same methods as in ERC20 standard
-    // we may falsely treat them as tokens, so in case we get any errors when getting token info
-    // we fallback to displaying custom transaction
-    isSendTokenTx = decimals !== null && symbol !== null
-  }
-
-  return isSendTokenTx
+  return !isSendERC721Transaction(tx, txCode, knownTokens) && isTokenTransfer(tx)
 }
 
 export const isERC721Contract = async (contractAddress: string): Promise<boolean> => {
   const ERC721Token = await getStandardTokenContract()
   let isERC721 = false
 
   try {
-    isERC721 = true
     await ERC721Token.at(contractAddress)
```

Comment:
This only checks that there is code not that the code is a ERC721 token, right? Also we are using a deprecated version of truffle-contracts, would be nice to update that to the new package (may in a different PR)